.github/ISSUE_TEMPLATE/server_installation_form.yaml

```diff
@@ -0,0 +1,88 @@
+name: Server installation issue
+description:  Report an issue with installing a server via nvim-lsp-installer
+labels: server-issue
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+          👋! This is not an issue template for questions! If you have questions, please refer to https://github.com/williamboman/nvim-lsp-installer/discussions/categories/q-a :)
+
+          Before filing an issue, make sure that you meet the minimum requirements mentioned in the README
+  - type: textarea
+    id: problem-description
+    attributes:
+      label: Problem description
+      description: A clear and concise description of what the issue is and why you think it's an issue with nvim-lsp-installer
+      validations:
+        required: true
+  - type: input
+    id: nvim-version
+    attributes:
+      label: Neovim version (>= 0.6)
+      description: Output of `nvim --version`
+      placeholder: |
+        NVIM v0.6.0-dev+209-g0603eba6e
+        Build type: Release
+        LuaJIT 2.1.0-beta3
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: nvim-lsp-installer version
+      description: commit hash, `git log --pretty=%h -1`
+      placeholder: 284e914
+  - type: input
+    attributes:
+      label: nvim-lspconfig version
+      description: commit hash, `git log --pretty=%h -1`
+      placeholder: 299e6395
+  - type: input
+    attributes:
+      label: Operating system/version
+      placeholder: macOS 11.5
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: Affected language servers
+      description: If this issue is specific to one or more language servers, list them here. If not, write 'all'.
+      placeholder: tsserver
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Steps to reproduce
+      description: Steps to reproduce using the minimal debug config.
+      placeholder: |
+        You can download it from here: https://github.com/williamboman/nvim-lsp-installer/blob/main/tests/minimal_debug_init.lua
+        1. `nvim -u ./minimal_debug_init.lua`
+        2. ...
```

Comment:
I feel like this would be better suited for the other issue type (the one that doesn't apply to the installation of servers). I feel like the process of installing servers is already pretty isolated, regardless of the surrounding environment, so being able to reproduce it using a minimal config isn't really that important, imo?

Reply:
Oh this is for the actual operation of the server, maybe server not launching, not attaching or another runtime issue that could be verified with a quick minimal setup. That's why I had the strict version checks in there.

I'll create a separate one for servers installation if you'd like. I didn't really consider that you had one specifically for the installation part.